security-privacy-questionnaire.md

```diff
@@ -28,26 +22,22 @@ This feature does not collect or expose any such information.
 
 ## 4. How does this specification deal with sensitive information?
 
-The only sensitive information this feature handles is an opaque string that can
-be mapped to a payment instrument by an issuing bank. This instrument ID is
-saved in the user agent along side the public key credential when the issuing
-bank creates a `PaymentCredential`. Later, an origin that has access to this
-instrument ID, presumably via a trusted server integration with the issuing
-bank, can provide it to the user agent via [Payment Request API] to exercise the
-corresponding `PaymentCredential`.
+The only sensitive information this feature handles is the public ID of the
+created credential. This credential ID is saved in the user agent along side
+the Relying Party ID when the issuing bank creates an SPC-enabled
+`PublicKeyCredential`. Later, an origin that has access to this credential ID,
+presumably via a trusted server integration with the issuing bank, can provide
+it to the user agent via [Payment Request API] to exercise the corresponding
+`PublicKeyCredential`.
 
 ## 5. Does this specification introduce new state for an origin that persists across browsing sessions?
 
-This feature allows an issuing bank to persist an icon, a descriptive string and
-an issuer-defined payment instrument ID with a public key credential when it
-creates a new `PaymentCredential`. However, this data is only used by the
-user agent to render UI to the user, and cannot be queried via JavaScript.
+No.
```

Comment:
Hmm, isn't the answer "Yes, but (1) we expect that to go away with discoverable credentials, and (2) the information is only available through the API." ?

Reply:
What state is that? A WebAuthn credential already persists across browsing sessions.

Reply:
Point taken. (I admit to not knowing the details about WebAuthn implementations.)